sdk/python/kfp/dsl/_container_op.py

```diff
@@ -1371,6 +1383,27 @@ def add_node_selector_constraint(self, label_name: str,
     super(ContainerOp, self).add_node_selector_constraint(label_name, value)
     return self
 
+  def add_cpu_request(self, cpu: str) -> 'ContainerOp':
+    """Adds a cpu request at runtime
```

Comment:
nit: summary line should end with a period.
https://google.github.io/styleguide/pyguide.html#381-docstrings